maestrowf/abstracts/enums/__init__.py

```diff
@@ -65,3 +65,11 @@ class State(Enum):
     TIMEDOUT = 10
     UNKNOWN = 11
     CANCELLED = 12
+
+
+class StudyStatus(Enum):
+    """Workflow status enumeration"""
+    FINISHED = 0
```

Comment:
For documentation purposes, it might be good to add inline comments here for the definitions of the enums. I plan to go back and update the other ones so their definitions are clear.

Reply:
So something like this: FINISHED = 0   # The Study has finished successfully, all steps ran